lib/client.js

```diff
@@ -493,6 +464,234 @@ class Client {
   prevPage(results) {
     return this.pagination.prevPage(results);
   }
+
+  /**
+   * Search for a FHIR resource, within a compartment, or the entire system
```

Comment:
Maybe this should say something like `with or without compartments` rather than `within a compartment`.

Reply:
Updated in a new commit